samcli/commands/package/exceptions.py

```diff
@@ -72,3 +72,12 @@ def __init__(self, template_file, ex):
         super(PackageFailedError, self).__init__(
             message=message_fmt.format(template_file=self.template_file, ex=self.ex)
         )
+
+
+class NoSuchBucketError(UserException):
+    def __init__(self, **kwargs):
+        self.kwargs = kwargs
+
+        message_fmt = "\n S3 Bucket does not exist."
```

Comment:
Space at start of line - right after `\n`.

Why did you remove the use `aws s3 mb` text from the message?

Reply:
`sam bootstrap` should come here logically.

Reply:
right. can you add that?